docs/en/stack/ml/df-analytics/ml-feature-importance.asciidoc

```diff
@@ -0,0 +1,35 @@
+[role="xpack"]
+[[ml-feature-importance]]
+=== Understanding results: {feat-imp}
+
+{feat-imp-cap} is calculated for supervised {ml} methods such as 
+<<dfa-regression,{regression}>> and <<dfa-classification,{classification}>>. 
+This value provides further insight into the results of a {dfanalytics-job} and 
+therefore helps interpret these results. As we mentioned, there are multiple 
```

Comment:
> As we mentioned...

I think this is referring to content that existed elsewhere.  I think this should be reworded or removed.